CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,159 @@
+# pyOpenSci Community Code of Conduct
+
+pyOpensci is dedicated to creating a welcoming, supportive and diverse 
+community around the open source Python tools that drive open
+science. We do this through a combination of community building, 
+mentorship, peer review of software and education and outreach. 
+
+We are building the pyOpenSci community in a way that promotes fairness 
+and justice for all particular focus on supporting those from 
+traditionally marginalized groups. We want every member of our community 
+to thrive; we want to make a positive impact on our community members work 
+and on the work of others. 
+
+### What this code of conduct defines 
+
+Our Code of Conduct defines expected behavior 
+and guidelines that help create such a community.
+
+Anyone who participates in a pyOpenSci activity either online or in person is expected to show respect and courtesy to others in all interactions. These interactions may occur in a variety of locations including:
+
+* in GitHub repositories
+* in our Slack channel
+* in our Discourse online forum 
+* during in-person events, 
+* when representing pyOpenSci in public, 
+* or in any event online or in person where pyOpenSci is involved.  
```

Comment:
"is involved" is vague. This can be ok, as long as we are prepared to handle situations in spaces run by another org where pyOpenSci is participating.

Reply:
oh shoot. i'm not sure . I was thinking say - we go to scipy and hold a BOF. it's a pyOpenSci session but scipy owns the meeting. or if we are representing pyopensci at a scientific python event.  but realistically can i uphold our COC at other events? probably not?

Reply:
TODO: possible reword - where pyOpenSci is leading the event. 
```suggestion
* or in any event online or in person where pyOpenSci is leading the event.  
```

Reply:
propose changing 'pyopensci is involved' to 'pyopensci is leading'